recipes/staircase/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "staircase" %}
+{% set version = "1.5.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ded45fb4444bb5273cee6ca13d092952a908c72968eef0c26776996adae4a1ec
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - matplotlib-base >3.0.0,<4
+    - numpy >=1.16.0,<2
+    - pandas >=0.24,<2
+    - python >=3.6,<4
+    - sortedcontainers >=2,<3
+
+test:
+  imports:
+    - staircase
+    - staircase.docstrings
+  source_files:
+    - tests
+  requires:
+    - python
```

Comment:
You don't have to list `python` under `requires` since it is pulled in as a `run` requirement.